components/compliance-service/ingest/server/compliance.go

```diff
@@ -37,18 +38,18 @@ type ComplianceIngestServer struct {
 	mgrClient          manager.NodeManagerServiceClient
 	automateURL        string
 	notifierClient     notifier.Notifier
-	updateManager      projectupdater.Manager
+	updateManager      *projectupdater.Manager
```

Comment:
[huge nit] totally out of scope of this PR, but I was expecting `projectUpdater` (and project-updater for the file name)

Reply:
Yeah, I agree with you, I don't like the mashing of 'project' and 'updater'. I remember this conversation a while ago when we merged all the projects to form the mono repo and we decided to follow the golang way from here https://blog.golang.org/package-names.